SPECS/ig/ig.signatures.json

```diff
@@ -0,0 +1,6 @@
+{
+ "Signatures": {
+  "ig-0.25.0-govendor-v1.tar.gz": "ba1c9153ee67cd9d1ebd31021ca3848c1099979c229b26bea6ad05470ac486ac",
+  "v0.25.0.tar.gz": "3d283e0f6e758b36e291e483e79e7a08c4ce2965a6c11f5c1ab671ced7015a6e"
```

Comment:
This line would also need to changed as `ig-0.25.0.tar.gz` since we changed the name in Source0.
A local build should help :)